### ISSUE TITLE
More animations (NiControllerSequences) in one nif, import ok, but export failure. 

### ISSUE THREAD

**Issue by olet** [BadDogSkyrim/PyNifly#244]
Import:  more animations in one Nif,  as NicontrollerManager - NiControllerSequence(s) , success, more animation-actions appear in blender. 

but export : NiControllerSequence(s) are lost, only one animation(only one current active action in blender was exported) , so failure. 

Can you fix that?  Thanks!

Test file : [Animated Dragon Wings](https://www.nexusmods.com/skyrim/mods/53271?tab=files)

**Comment by BadDogSkyrim** [comment]
Yes, this is actually what I'm working on right now. The latest will do import/export of the noble chest open/close animations, but it's not complete.
